tests/Behat/DrupalContext.php

```diff
@@ -62,4 +62,22 @@ public function assertFieldsNotPresent($fields): void {
     }
   }
 
+  /**
+   * Checks that a given image is present in the page.
```

Comment:
... on the page. ?

Reply:
I think we used "in" to infer the image is inside the page.